testing/supporting_docs/appium_information.md

```diff
@@ -0,0 +1,69 @@
+## Appium 
+Appium is a cross platform UI testing tool for mobile applications running on 
+either android or iOS devices.  It can be used to test native applications, 
+hybrid applications like Cordova and Xamarin, and web app testing through mobile 
+device browsers.  One of major points of the Appium philosophy is to avoid time 
+consuming processes like compiling to run UI tests.
+
+
+### Appium Locator/Selector Strategies
+
+![alt text](./images/appium_selector_strategies.png "Appium Selector Strategies")
+https://appium.io/docs/en/commands/element/find-elements/#selector-strategies
+
+The picture above shows the available selector strategies to appium in order to 
+locate elements in a mobile application. Note that some of the descriptions also 
+shows the required attributes in for both Android and iOS applications.  It can 
+be a problem when using cross platform tools such as Cordova to use these 
+selectors and the webview context should be used.  It is important to note that 
+some clients do not offer all of the selector strategies as shown in the table 
+above.
+
+## Appium Clients
+Appium uses a client-server architecture that interacts with the emulator or browser 
+when running UI test.  The Appium client is used by the test code to create a session 
+with the Appium server and will send the instructions to the server which will then 
+interact with the emulator, simulator or connected device to perform the UI tests.
+
+One of the most important sections of code when writing tests is the specification of 
+‘desired capabilities’.  These ‘desired capabilities’ instruct the server as to the 
+type of session required and various capabilities which will be available to the 
+server during the test runs.  For example, they can be used to block pop-ups or 
+security warnings in browsers or to check if an expected version of Android or iOS 
```

Comment:
@twnolan consistency should be persisted with the word 'android' across the whole document, e.g. starting with either lower or upper case.